samples/sleeper/Makefile

```diff
@@ -16,6 +17,9 @@ PROGS = sleeper
 
 all: $(PROGS)
 
+install: all
+	@echo "--- The example app $(PROGS) is not installed ---"
```

Comment:
Such kind of message will mislead the user, (s)he will decide that smth wrong happened during installation and it should be fixed somehow.